docs/changes.rst

```diff
@@ -154,6 +154,121 @@ Compatibility notes:
   Since the "trick" does not work if BORG_PASSPHRASE is set, this does generally
   not affect scripts.
 
+New features:
+
+- integrity checking for important files used by borg:
+
+  - repository: index and hints files
+  - cache: chunks and files caches, archive.chunks.d
+  - temporary metadata cache contents
+- Verify most operations against securitymanager, #2487
+- use assert_secure for all commands that use the manifest
```

Comment:
165 is redundant to 164. Perhaps rather mention the effect, i.e. location, manifest timestamp and key types are now checked for almost all non-debug commands.